packages/gen-typescript-declarations/scripts/update-goldens.js

```diff
@@ -29,32 +29,70 @@ const {generateDeclarations} = require('../lib/gen-ts');
 
 const fixturesDir = path.join(__dirname, '..', 'src', 'test', 'fixtures');
 const goldensDir = path.join(__dirname, '..', 'src', 'test', 'goldens');
+const googGoldensDir = path.join(__dirname, '..', 'src', 'test', 'goldens_goog');
 
 const filter = process.env.UPDATE_GOLDENS_FILTER || '';
 
 if (!filter) {
   fsExtra.emptyDirSync(goldensDir);
 }
 
-for (const fixture of fs.readdirSync(fixturesDir)) {
-  if (filter && !fixture.includes(filter)) {
-    continue;
+async function main() {
+  const promises = [];
+  for (const fixture of fs.readdirSync(fixturesDir)) {
+    promises.push((async () => {
+      if (filter && !fixture.includes(filter)) {
+        return;
+      }
+      fsExtra.emptyDirSync(path.join(goldensDir, fixture));
+
+      let config = {};
+      const configPath = path.join(fixturesDir, fixture, 'gen-tsd.json');
+      if (fs.existsSync(configPath)) {
+        config = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+      }
+      config.hideWarnings = true;
+
+      const declarations = await generateDeclarations(
+          path.join(fixturesDir, fixture), config);
+
+      for (const [filename, contents] of declarations) {
+        const outPath = path.join(goldensDir, fixture, filename);
+        fsExtra.mkdirsSync(path.dirname(outPath));
+        fs.writeFileSync(outPath, contents);
+      };
+    })());
   }
-  console.log('making goldens for ' + fixture);
-  fsExtra.emptyDirSync(path.join(goldensDir, fixture));
 
-  let config = {};
-  const configPath = path.join(fixturesDir, fixture, 'gen-tsd.json');
-  if (fs.existsSync(configPath)) {
-    config = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+  for (const fixture of fs.readdirSync(fixturesDir)) {
```

Comment:
Same here, could just have run the generation/saving code twice instead of copying the whole loop.

Reply:
See above